_includes/master/charts/calico/values.yaml

```diff
@@ -1,4 +1,39 @@
-typha:
-  enabled: false
+# Select which datastore mode. Can be 'etcd' or 'kdd'.
+# Note: Changing this after installation is not supported. 
+datastore: kdd
+
+# Sets the networking mode. Can be 'calico', 'flannel', or 'none'
 network: "none"
+
+# Global docker image registry. This will set the image registry for all images *except* flannel.
+imageRegistry: "calico/"
```

Comment:
Wait, remind me what this is used for if we declare the full images below?

We don't use `calico/calico/node`, I assume :D

Reply:
whoops :+1:

Reply:
Can we remove this now?

Reply:
Hrm, this doesn't seem to be working the way it was in my head.

I thought we were going to have values.yaml specify precisely the images (including registry) that we wanted to install. It would pull that registry from the per-version config specified here: https://github.com/projectcalico/calico/blob/master/_config.yml#L54

And, eventually that per-version config would be removed altogether and we'd simply have versions.yml which specifies the image name in its entirety, including registry.

If users wanted to modify this themselves, they'd just update the image names directly.